scripts/data-lint.js

```diff
@@ -0,0 +1,50 @@
+const inspect = require("util").inspect;
+const { join, resolve } = require("path");
+
+const {
+	communitySchema,
+	demosSchema,
+	pluginsSchema,
+	sitesSchema,
+	startersSchema
+} = require("./schemas");
+
+const { schemaLint } = require("./data-lint-lib");
+
+const dataDir = resolve(join(__dirname, "..", "src", "_data"));
+let errors = [];
+
+main();
+
+async function main() {
+	await Promise.all([
+		dataLint(join(dataDir, "community"), "*.js", communitySchema),
+		dataLint(join(dataDir, "demos"), "*.js", demosSchema),
+		dataLint(join(dataDir, "plugins"), "*.json", pluginsSchema),
+		dataLint(join(dataDir, "sites"), "*.json", sitesSchema),
+		dataLint(join(dataDir, "starters"), "*.json", startersSchema)
+	]);
+
+	if (errors.length) {
+		console.error(inspect(errors, { depth: 4 }));
+		process.exitCode = 1;
+	}
+
+	console.info("All data files are clean!");
```

Comment:
I would like to give some sort of feedback (somewhere) about a successful run.

Reply:
Hhmm. I think this needs to be in an `else` or maybe the previous `if` needs a `return` statement so we don't `console.error(...)` a bunch of errors then write "All data files are clean!" to STDOUT.

Reply:
A `process.exitCode` should terminate the program, shouldn't it?

Reply:
I believe that `process.exitCode = 1` just sets an exit code, but continues execution. 

> A number which will be the process exit code, when the process either exits gracefully, or is exited via `process.exit()` without specifying a code.
> &mdash; via https://nodejs.org/api/process.html#process_process_exitcode

If you want to exit immediately and stop further code execution, you need to use [`process.exit(1)`](https://nodejs.org/api/process.html#process_process_exitcode) instead, I believe.

Reply:
Sounds about right.
I'm still not sure, whether we are at the right place here or whether that should happen in the lib file instead.
The more I think about it the more I feel it should move.

Reply:
I guess it depends on if you want it to say **"All data files are clean!"** once, or if you move it into the lib, it'd potentially say **"All /Volumes/Dev/github/11ty-website/src/_data/community/ files are clean!"** for each directory separately. For example "community", "demos", "plugins", "sites", and "starters" directories.

We could also try splitting the directory name and just returning the last directory or three if you wanted to limit the output to "community" or "src/_data/community".